.eslintignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+build
```

Comment:
new line please

Reply:
@tyrro Removed in [f7908f4](https://github.com/manh-t/react-survey/pull/39/commits/f7908f44155a599551ebd98cc0f9b826151d6483)